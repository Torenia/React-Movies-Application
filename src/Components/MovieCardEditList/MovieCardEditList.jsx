import styled from 'styled-components';

export const MovieCardEditList = styled.div`
  position: absolute;
  top: 15px;
  right: 15px;
  width: 170px;
  height: 115px;
  background-color: #232323;
  border-radius: 5px;
  
  ul {
    list-style-type: none;
    padding: 0;
    margin: 35px 0 0;
  
    li {
      padding: 10px 0;
      padding-left: 20px;
      border-radius: 5px;
  
      &:hover{
        background-color: #f65261;
      }
  
       a {
         color: #fff;
         text-decoration: none;
        } 
    }
  }
  
  button {
    position: absolute;
    top: 0;
    right: 0;
    font-size: 15px;
    background: #232323;
    width: 35px;
    height: 35px;
    border: none;
    border-radius: 50%;
    outline:none;
    cursor: pointer;
  
    &:focus {
      outline:none;
    }
  
    &:first-child:after {
       content: "\\2716";
       display: inline-block;
       color: #fff;
    }
`;
