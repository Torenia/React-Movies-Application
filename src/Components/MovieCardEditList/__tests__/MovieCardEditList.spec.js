import React from 'react';

import { MovieCardEditList } from '../MovieCardEditList';

describe('<MovieCardEditList />', () => {
  let wrapper;

  beforeEach(() => {
    wrapper = shallow(<MovieCardEditList />);
  });

  it('should render a movie card edit list', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
