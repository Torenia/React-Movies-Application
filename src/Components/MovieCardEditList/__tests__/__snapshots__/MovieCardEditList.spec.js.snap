// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<MovieCardEditList /> should render a movie card edit list 1`] = `
.c0 {
  position: absolute;
  top: 15px;
  right: 15px;
  width: 170px;
  height: 115px;
  background-color: #232323;
  border-radius: 5px;
}

.c0 ul {
  list-style-type: none;
  padding: 0;
  margin: 35px 0 0;
}

.c0 ul a {
  color: #fff;
  -webkit-text-decoration: none;
  text-decoration: none;
}

.c0 ul li {
  padding: 10px 0;
  padding-left: 20px;
  border-radius: 5px;
}

.c0 ul li:hover {
  background-color: #f65261;
}

.c0 button {
  position: absolute;
  top: 0;
  right: 0;
  font-size: 15px;
  background: #232323;
  width: 35px;
  height: 35px;
  border: none;
  border-radius: 50%;
  outline: none;
  cursor: pointer;
}

.c0 button:focus {
  outline: none;
}

.c0 button:first-child:after {
  content: "\\2716";
  display: inline-block;
  color: #fff;
}

<div
  className="c0"
/>
`;
